File: test_decode.js
const base64 = 'eyJwYXNzd29yZCI6IjIwMjYiLCJyZWRlZW1Db2RlIjoiIiwicmVkcGFja2V0TGluayI6Imh0dHBzOi8vdXIuYWxpcGF5LmNvbS9fMzZHbkNLYzZudlpDU0JZcXdaZW80MCIsImtvdWxpbmciOiIyMDI2bmV3eWVhciIsInRpdGxlIjoi5rOh5rOh6bK456Wd5oKo5paw5bm05b+r5LmQIiwiYmxlc3NpbmciOiLpmaTlpJXlv6vkuZAtLea0siJ9';

function base64Decode(base64) {
    try {
        return decodeURIComponent(atob(base64).split('').map(c => '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2)).join(''));
    } catch (e) {
        console.error('Base64解码失败:', e);
        return null;
    }
}

const decoded = base64Decode(base64);
console.log('解码结果:', decoded);

const config = JSON.parse(decoded);
console.log('暗号:', config.password);

File: test_decode.py
import base64
import json
import urllib.parse

base64_str = 'eyJwYXNzd29yZCI6IjIwMjYiLCJyZWRlZW1Db2RlIjoiIiwicmVkcGFja2V0TGluayI6Imh0dHBzOi8vdXIuYWxpcGF5LmNvbS9fMzZHbkNLYzZudlpDU0JZcXdaZW80MCIsImtvdWxpbmciOiIyMDI2bmV3eWVhciIsInRpdGxlIjoi5rOh5rOh6bK456Wd5oKo5paw5bm05b+r5LmQIiwiYmxlc3NpbmciOiLpmaTlpJXlv6vkuZAtLea0siJ9'

# Decode base64
decoded_bytes = base64.b64decode(base64_str)
decoded_str = decoded_bytes.decode('utf-8')

print('解码结果:', decoded_str)

# Parse JSON
config = json.loads(decoded_str)
print('暗号:', config['password'])
print('完整配置:', json.dumps(config, ensure_ascii=False, indent=2))
